.gitignore

```diff
@@ -26,6 +26,7 @@ dist/
 # Miscellaneous files
 *.sw[op]
 *.DS_Store
+__debug_bin*
```

Comment:
```suggestion
```

Reply:
@djaglowski These were building up in my directory as I've been debugging and showing up as files that could be committed. Any reason not to ignore them?